producers/http/http.go

```diff
@@ -126,6 +128,22 @@ func (p *producerImpl) writeObjectToStore(d producers.Datapoint, m producers.Met
 	p.store.Set(qualifiedName, newMessage)
 }
 
+// sortTags turns a map[string]string into a slice of key/values, sorted by key.
+func sortTags(tags map[string]string) [][]string {
```

Comment:
You could also return a slice of 2-element arrays from this function  (`[][2]string`).